osu.Game/Rulesets/Mods/Mod.cs

```diff
@@ -69,7 +69,7 @@ public abstract class Mod : IMod, IJsonSerializable
         /// <summary>
         /// Creates a copy of this <see cref="Mod"/> initialised to a default state.
         /// </summary>
-        public virtual Mod CreateCopy() => (Mod)Activator.CreateInstance(GetType());
+        public virtual Mod CreateCopy() => (Mod)MemberwiseClone();
```

Comment:
I'm still not quite sure about this. I believe in its current state it will break the easy mod.

What we really want here is to bind bindables between mods, so perhaps a custom implementation that _does_ use `CreateInstance` and also reflection to bind (or copy? unsure) bindables.

Reply:
Actually it probably won't break the easy mod since it's cloned with a zero-state, still feels kind of dodgy though.